compose/ui/ui/src/jsWasmMain/kotlin/androidx/compose/ui/window/ComposeWindow.js.kt

```diff
@@ -191,12 +191,17 @@ private class ComposeWindow(
     fun resize(newSize: IntSize) {
         canvas.width = newSize.width
         canvas.height = newSize.height
-        _windowInfo.containerSize = IntSize(canvas.width, canvas.height)
+        onChangeWindowBounds()
         layer.layer.attachTo(canvas)
         layer.setSize(canvas.width, canvas.height)
```

Comment:
To be consistent, and to avoid bugs, we need to pass `canvas.width * density` to:
1. `layer.setSize`
2. `scene.sendPointerEvent`

Reply:
@igordmn No, we should not. Let's sync offline

Reply:
@igordmn it looks like we should indeed discuss this because the misunderstanding and confusion of different options can be actually disastrous